src/utils/commands.ts

```diff
@@ -165,10 +165,16 @@ function attachOptions(
 }
 
 export async function getUsernameParam(interaction: CommandInteraction) {
+  const discordTagRegex = /<@!?(\d+)>/;
   const username = interaction.options.getString('username', false);
-  if (username) return username;
+  const isDiscordId = username.match(discordTagRegex);
```

Comment:
This doesnt error when no username is passed as an option?

Reply:
fixed. weird TS inference